tools/internal_ci/linux/grpc_xds_k8s_run_test.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## xDS test server/client Docker images
+readonly IMAGE_REPO="gcr.io/grpc-testing/xds-interop"
+
+find_latest() {
+  gcloud container images list-tags --filter=tags\~v1\.\\d\\d\.x "${IMAGE_REPO}/${1}-${2}" '--format=table[no-heading](tags)' | awk -F, '{print $2}' | sort | tail -n 1
+}
+
+if [ "${LATEST_BRANCH}" == "" ]; then
+  cpp_server=$(find_latest cpp server)
+  cpp_client=$(find_latest cpp client)
+  go_server=$(find_latest go server)
+  go_client=$(find_latest go client)
+  java_server=$(find_latest java server)
+  java_client=$(find_latest java client)
+  LATEST_BRANCH="$((echo $cpp_server; echo $cpp_client; echo $go_server; echo $go_client; echo $java_server; echo $java_client;) | sort -g | head -1)"
+fi
+
+if [ "${OLDEST_BRANCH}" == "" ]; then
+  OLDEST_BRANCH="v1.$(expr $(echo ${LATEST_BRANCH} | cut -f 2 -d.) - 9).x"
```

Comment:
Shellcheck also complains about this:

```
In grpc_xds_k8s_run_test.sh line 34:
  OLDEST_BRANCH="v1.$(expr $(echo ${LATEST_BRANCH} | cut -f 2 -d.) - 9).x"
                      ^--^ SC2003 (style): expr is antiquated. Consider rewriting this using $((..)), ${} or [[ ]].
                           ^-- SC2046 (warning): Quote this to prevent word splitting.
                                  ^--------------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
  OLDEST_BRANCH="v1.$(expr $(echo "${LATEST_BRANCH}" | cut -f 2 -d.) - 9).x"
```

Reply:
done